mk/llvm.mk

```diff
@@ -83,7 +83,7 @@ endif
 LLVM_LINKAGE_PATH_$(1):=$$(abspath $$(RT_OUTPUT_DIR_$(1))/llvmdeps.rs)
 $$(LLVM_LINKAGE_PATH_$(1)): $(S)src/etc/mklldeps.py $$(LLVM_CONFIG_$(1))
 	$(Q)$(CFG_PYTHON) "$$<" "$$@" "$$(LLVM_COMPONENTS)" "$$(CFG_ENABLE_LLVM_STATIC_STDCPP)" \
-		$$(LLVM_CONFIG_$(1))
+	        "$$(CFG_ENABLE_LLVM_LIBCPP)" $$(LLVM_CONFIG_$(1))
```

Comment:
In the configure script it looks like this is `CFG_ENABLE_LIBCPP` instead of `LLVM_LIBCPP`, was this defined elsewhere though?